newIDE/app/src/AssetStore/ExtensionStore/ExtensionStoreContext.js

```diff
@@ -39,6 +42,9 @@ export const ExtensionStoreContext = React.createContext<ExtensionStoreState>({
   error: null,
   searchText: '',
   setSearchText: () => {},
+  allCategories: [],
+  chosenCategory: '',
```

Comment:
Should it be null so that it's clear nothing has been chosen?

Reply:
Alright. I don't know if it needs a comment then here

Reply:
I added one.